File: 0x01-python-if_else_loops_functions/6-print_comb3.py
#!/usr/bin/python3
for FirstDigit in range(0, 8):
    for SecondDigit in range(FirstDigit + 1, 10):
        print("{}".format(FirstDigit), end="")
        print("{}".format(SecondDigit), end=", ")
print("{:d}{:d}".format(FirstDigit + 1, SecondDigit))

File: 0x01-python-if_else_loops_functions/6-print_comb3.c
#include <stdio.h>

int main(void)
{
	int FirstDigit, SecondDigit;

	for (FirstDigit = 0; FirstDigit < 10; FirstDigit++)
	{
		for (SecondDigit = FirstDigit + 1; SecondDigit < 10; SecondDigit++)
		{
			printf("%d%d, ", FirstDigit, SecondDigit);
		}
	}
	putchar('\n');
	return (0);
}
